File: java/tempo_de_jogo.java
import java.util.Locale;
import java.util.Scanner;

public class tempo_de_jogo {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int hinicial, hfinal;

		System.out.print("Hora inicial: ");
	    hinicial = sc.nextInt();

	    System.out.print("Hora final: ");
	    hfinal = sc.nextInt();

	    if (hinicial < hfinal) {
	    	System.out.printf("O JOGO DUROU %d HORA(S)\n", hfinal - hinicial);
	    }
	    else {
			System.out.printf("O JOGO DUROU %d HORA(S)\n", 24 - (hinicial - hfinal));
	    }

		sc.close();
	}
}

File: c/tempo_de_jogo.c
#include <stdio.h>

int main(){
    int hinicial, hfinal;

    printf("Hora inicial: ");
    scanf("%d", &hinicial);

    printf("Hora final: ");
    scanf("%d", &hfinal);

    if (hinicial < hfinal) {
        printf("O JOGO DUROU %d HORA(S)\n", hfinal - hinicial);
    }
    else {
		printf("O JOGO DUROU %d HORA(S)\n", 24 - (hinicial - hfinal));
    }

	return 0;
}
